constructor/osx/update_path.sh

```diff
@@ -18,15 +18,23 @@ INIT_FILES=$("$PREFIX/bin/python" -m conda init --all | tee)
 # have the correct owner.
 if [[ "${USER}" != "root" ]]; then
     echo "Fixing permissions..."
-    read -r -a MODIFIED_FILES <<< "$(\
+    MODIFIED_FILES=()
+    while read -r line; do
+        MODIFIED_FILES+=("$line")
+    done <<< "$(\
       echo "${INIT_FILES}" |\
-      awk '/modified/{print $2}' |\
+      grep -E "^modified" |\
+      sed -e 's/^modified */' |\
```

Comment:
Hm, don't we need a second clause for the replacement in `s/`? i.e. 
```suggestion
      sed -e 's/^modified *//' |\
```

Reply:
Yes, it looks like there was a copy-paste error from my virtual machine that I used for testing. I will double check that things are identical before re-requesting a review.